src/swaps/oneinch/OneinchSwapProvider.ts

```diff
@@ -238,6 +238,11 @@ class OneinchSwapProvider extends SwapProvider {
     return null;
   }
 
+  // @ts-ignore
```

Comment:
Why the ts-ignore everywhere? going to need a good reason

Reply:
It's probably to ignore the unused parameter. Better use `_` for the parameter name to ignore the warning, i.e. `_quoteRequest`

Reply:
Resolved  ts-ignore removed and _ used